INSTALL.md

```diff
@@ -6,9 +6,13 @@ Build Requirements
 
 To compile Coq yourself, you need:
 
-- [OCaml](https://ocaml.org/) (version >= 4.05.0)
+- [OCaml](https://ocaml.org/) (version >= 4.05.0, < 4.12.0)
```

Comment:
```suggestion
- [OCaml](https://ocaml.org/) (4.05.0 <= version < 4.12.0)
```

If you like.  Fine either way.

Reply:
It's more mathematical but less readable (?), so I don't know.